phpunit/blocks/render-block-media-text-test.php

```diff
@@ -94,21 +87,66 @@ public function test_render_block_core_media_text_featured_image() {
 		$rendered   = gutenberg_render_block_core_media_text( $attributes, $content );
 		$this->assertStringContainsString( 'background-image:url(' . wp_get_attachment_image_url( self::$attachment_id, 'full' ) . ')', $rendered );
 		$this->assertStringNotContainsString( '<img', $rendered );
+	}
+
+	/**
+	 * Test gutenberg_render_block_core_media_text with the featured image on the left,
+	 * and a second media & text block nested inside the content area.
+	 */
+	public function test_render_block_core_media_text_featured_image_nested() {
 
-		$rendered = gutenberg_render_block_core_media_text( $attributes, $content_nested );
+		global $wp_query;
+
+		// Fake being in the loop.
+		$wp_query->in_the_loop = true;
+		$wp_query->post        = self::$post;
+
+		$wp_query->posts = array( self::$post );
+		$GLOBALS['post'] = self::$post;
```

Comment:
I want to try to see if I can avoid repeating this code but I might not have time until the end of June.

Reply:
There are only three tests, so I reckon it's fine to iterate. 

Thanks a lot for splitting them up!